File: src/test/java/br/com/jiratorio/config/WireMockConfig.java
package br.com.jiratorio.config;

import com.github.tomakehurst.wiremock.WireMockServer;
import com.github.tomakehurst.wiremock.client.WireMock;
import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class WireMockConfig {

    @Bean
    public WireMockServer wireMockServer() {
        WireMockServer wireMockServer = new WireMockServer(
                WireMockConfiguration
                        .options()
                        .port(8888));
        wireMockServer.start();

        WireMock.configureFor(wireMockServer.port());

        return wireMockServer;
    }

}

File: test-library/src/main/kotlin/br/com/jiratorio/WireMockConfig.kt
package br.com.jiratorio

import com.github.tomakehurst.wiremock.WireMockServer
import com.github.tomakehurst.wiremock.client.WireMock
import com.github.tomakehurst.wiremock.core.WireMockConfiguration
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class WireMockConfig {

    @Bean
    fun wireMockServer(): WireMockServer {
        val wireMockServer = WireMockServer(
            WireMockConfiguration
                .options()
                .port(8888)
        )
        wireMockServer.start()

        WireMock.configureFor(wireMockServer.port())

        return wireMockServer
    }

}
